Cinema/routes/moviesRoutes.ts

```diff
@@ -1,59 +1,43 @@
 import { Router } from "express";
 import * as moviesBLL from "../BLL/moviesBLL";
 import * as authMiddlewares from "../middlewares/authMiddlewares";
+import { UserPermissions } from "../types/objectTypes";
 const router = Router();
 
 //Endpoint localhost:3000/movies
 
-// Navigation
-
-router.get(
-  "/add-movie",
-  authMiddlewares.jwtMiddleware,
-  authMiddlewares.isAuth,
-  moviesBLL.getAddMoviePage
-);
-router.get(
-  "/edit-movie/:movieId",
-  authMiddlewares.jwtMiddleware,
-  authMiddlewares.isAuth,
-  moviesBLL.getEditMoviePage
-);
-
-// CRUD - Create, Read, Update, Delete
-
 router.get(
   "/",
   authMiddlewares.jwtMiddleware,
```

Comment:
I am not sure but I believe there is a way to define a middleware for all the routes instead of defining it for each one we are creating